app/commands/v2/put_content.rb

```diff
@@ -1,6 +1,7 @@
 module Commands
   module V2
     class PutContent < BaseCommand
+      ITEM_NOT_FOUND = Class.new
```

Comment:
This caught me as a bit of WTF at first, but I see what you did here and it's elegant 👍